package.json

```diff
@@ -63,7 +63,7 @@
     "nyc": "^15.0.0"
   },
   "engines": {
-    "node": ">=10.18.0"
+    "node": ">=12.16.3"
```

Comment:
Should we use `>=12.20.0` so we can use ES modules natively?

Reply:
I think it would be fine. That I know of this is only being used via `buildbot` and there are no plans to change that in a near future.

Reply:
I used `12.16.3` since it's the buildbot Node.js system version (unless I'm looking in the [wrong place](https://github.com/netlify/buildbot/blob/b7aeb2675f82192daa8545b22bebab9c3b179d0e/Dockerfile#L7)).

If `12.20.0` is required for `get-bin-path` we should wait with this breaking change as it won't allow us to upgrade that dependency.

Reply:
I don't think it's super urgent to do this now.

We can wait until we bump the system Node.js version

Reply:
:+1: